tag-rule/nitagrule.yml

```diff
@@ -0,0 +1,579 @@
+            required:
+              - searchPath
+              - tagDataType
+              - conditions
+            properties:
+              searchPath:
+                description: A string describing which tag(s) a rule applies to. This can take the form of either
+                  a single tag's path, or a string containing wildcard characters which can match multiple tags.
+                type: string
+                example: '*.System.Health.DiskUsePercentage'
+              tagDataType:
+                $ref: '#/definitions/TagDataType'
+              conditions:
+                $ref: '#/definitions/Conditions'
+              disabled:
+                description: A value indicating whether or not a rule is disabled.
+                type: boolean
+                example: false
+                default: false
+              displayName:
+                description: Display string for a rule's name.
+                type: string
+                example: Low Disk Space Rule
+              description:
+                description: Display string for a rule's description.
+                type: string
+                example: Preconfigured rule for monitoring managed systems' disk space
+              alarmInstanceDisplayNameTemplate:
+                description: Template string which will be used to set the *displayName* field on alarms created
+                  by a rule. Values surrounded by angle brackets will be substituted with values from the tag's property
+                  dictionary. For example, if an *alarmInstanceDisplayNameTemplate* contained the string "Excessive vibration
+                  on turbine <turbine_number>", and the rule was evaluated against a tag whose property dictionary
+                  contained the key "turbine_number" with value "5", then the rule would request that an alarm with 
+                  display name "Excessive vibration on turbine 5" be created.
+                example: Excessive vibration on turbine <turbine_number>
+                type: string
+              alarmInstanceDescriptionTemplate:
+                description: Template string which will be used to set the *description* field on alarms created by a rule.
+                  Values surrounded by angle brackets will be substituted with values from the tag's property dictionary.
+                  See the description of AlarmInstanceDisplayNameTemplate for more information.
+                example: Disk space on <system> is low. Disk usage was <value>% when this alarm was triggered.
+                type: string
+              keywords:
+                $ref: '#/definitions/Keywords'
+              properties:
+                $ref: '#/definitions/Properties'
+      responses:
+        200:
+          description: OK
+          schema:
+            type: object
+            title: CreateRuleResponse
+            properties:
+              id:
+                description: The ID of the created rule
```

Comment:
Should we mention whether it's unique or not here?

Reply:
I think most users will assume that IDs are unique, but it's OK to call it out here to dispel confusion. We also mention that it's unique on the definition of the ID field on the rule type.